pylint_pytest/checkers/fixture.py

```diff
@@ -91,6 +104,42 @@ def visit_module(self, node):
             # restore output devices
             sys.stdout, sys.stderr = stdout, stderr
 
+    def visit_decorators(self, node):
+        """
+        Walk through all decorators on functions.
+        Tries to find cases:
+            When uses `@pytest.fixture` with `scope` as positional argument (deprecated)
+            https://docs.pytest.org/en/stable/reference.html#pytest-fixture
+                >>> @pytest.fixture("module")
+                >>> def awesome_fixture(): ...
+                Instead
+                >>> @pytest.fixture(scope="module")
+                >>> def awesome_fixture(): ...
+            When uses `@pytest.mark.usefixtures` for fixture (useless because didn't work)
+            https://docs.pytest.org/en/stable/fixture.html#usefixtures
+                >>> @pytest.mark.usefixtures("another_fixture")
+                >>> @pytest.fixture
+                >>> def awesome_fixture(): ...
+        Parameters
+        ----------
+        node : astroid.scoped_nodes.Decorators
+        """
+        uses_fixture_deco, uses_usefixture_deco = 0, 0
```

Comment:
I'd prefer we don't use `int` for these two variables. We're not using the count anywhere in the below code and they can be replaced with `bool` and
```python
uses_fixture_deco |= _is_pytest_fixture(decorator)
uses_usefixture_deco |= _is_pytest_mark_usefixtures(decorator)
```